### PR TITLE
Go to safe_room at the end of hunting-buddy

### PR DESCRIPTION
**Pull request by kamarcum**
It does what you think it does.

### CODE REVIEW

File: hunting-buddy.lic [hunk 1]
```diff
@@ -39,7 +39,7 @@ class HuntingBuddy
       fput('release hyh') if DRSpells.active_spells['Hydra Hex']
       fput('release ee') if DRSpells.active_spells['Electrostatic Eddy']
     end
-    walk_to(793)
+    walk_to(@settings.safe_room)
```
Comment (rpherbig) [inline hunting-buddy.lic]: Line 49 should also be changed.

**Comment by rpherbig** [comment]
I'd go ahead and update `pick` in this PR.

Do you think line 32 of `crossing-training` needs to change? It provides a default value for `settings.outdoor_room`.

**Comment by kamarcum** [comment]
> I'd go ahead and update pick in this PR.

I find that less compelling at the moment. The part of `pick` that goes to 793 is also buying lockpicks from a shop in crossing. So that chunk of logic would need more work to become city-agnostic.

File: hunting-buddy.lic (hunk 1 above)
Comment (kamarcum) [inline hunting-buddy.lic]: Ah good find.

**Comment by rpherbig** [comment]
> I find that less compelling at the moment. The part of pick that goes to 793 is also buying lockpicks from a shop in crossing. So that chunk of logic would need more work to become city-agnostic.

It's also one less thing for us to forget when we put in that work later.

**Comment by rcuhljr** [comment]
The only thing that pick needed to do was step outside of the store when done buying. just replace it with a move('out') and be done.

**Pull request merged**